.github/ISSUE_TEMPLATE/requirement.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Requirement
+about: Use this template to propose an LWS requirement.
+title: "[REQ] <<brief description of requirement>>"
+labels: triage
+
+---
+
+**A [component],**
+**[MUST,SHOULD,MAY...] <<description or requirement>>**
+
+NOTE
+<< additional notes>>
+
+
+
+  The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+  NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+  "OPTIONAL" in this document are to be interpreted as described in
+  RFC 2119.
+
+[RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt)
```

Comment:
```suggestion
  The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
  NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
  "OPTIONAL" in this document are to be interpreted as described in
  RFC 2119.

[RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt)
```
As discuss, requirements are not conformance clauses (and anyway, the UCR document is non-normative)

Reply:
Right. I would just add this because it came up in a meeting: Irrespective of the UCR document being "non-normative" at a top level, as far as interoperable implementations go, the language within - whether it follows established conventions - is important to not conflate. People arrive at these documents with different perspectives. It is also partly about how well it communicates as well as helping the reader either familiarise themselves with the language or recognise something they are familiar with.

Regarding the point about RFC keywords for requirement levels, I haven't seen any UCR document using them, but perhaps that's just me, and so I'm open to seeing any evidence...

Reply:
I accepted the suggestion, although, imho, it makes things more confusing, not less. Knowing that this is a non-normative document, it would be far easier to keep the meaning defined by rfc 2119, agree on the intent of the requirement and then the only thing we had to so was to copy the already agreed upon text in the normative document. Any reader would have been familiar with the intent already and, if we found reasons to change it, we would have and maintain consistency between the normative doc and this one. This way we'll have to have the conversation not once, but twice.